tests/test_objective_funs.py

```diff
@@ -1661,3 +1662,26 @@ def test_objective_no_nangrad():
         obj.build()
         g = obj.grad(obj.x(eq))
         assert not np.any(np.isnan(g)), str(objective)
+
+
+@pytest.mark.unit
+def test_asymmetric_normalization():
+    """Tests normalizations for asymmetric equilibrium."""
+    # related to PR
+    a = 0.6
```

Comment:
Could there be a potential code coverage issue if a is not close to 0. 
The if condition in the fix that we discussed is used when a is close to 0.
If a is finite that if condition is not called.

Reply:
the if condition is not on `eq.compute("a")`, it is on `R10` and `Z10` which are zero in this example